redhat-developer/web-terminal-tooling
# Update knative CLI to v1.11.2

KNative v1.11.2 was just released yesterday on March 18th (both [upstream](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/serverless/1.11.2/) and [downstream](https://access.redhat.com/downloads/content/openshift-serverless-clients/1.11.2-4.el8/x86_64/fd431d51/package)), so I decided to include it in the WTO 1.10 release since the release is not done yet.

# Testing
I've pushed the image built from this PR to `quay.io/aobuchow/wto-tooling:1.10-2` for testing.

1. Launch an OpenShift 4.15 cluster
2. Install the web-terminal operator
3. Create a web terminal and click the link to specify a custom image, and provide `quay.io/aobuchow/wto-tooling:1.10-2`. Alternatively, create a web terminal with the default image and run `wtoctl set image quay.io/aobuchow/wto-tooling:1.10-2` in the web terminal console
4. Verify the web terminal starts up successfully. Run `kn version` or `help` and verify that the version of knative installed is v1.11.2
